dsl/src/main/scala/com.crobox.clickhouse/dsl/language/LogicalFunctionTokenizer.scala

```diff
@@ -7,39 +7,37 @@ trait LogicalFunctionTokenizer {
 
   def tokenizeLogicalFunction(col: LogicalFunction)(implicit ctx: TokenizeContext): String =
     (col.left.asOption, col.right.asOption) match {
-      case (None, None)        => "1"
-      case (Some(left), None)  => tokenizeColumn(left)
-      case (None, Some(right)) => tokenizeColumn(right)
+      case (None, None)       => "1"
+      case (Some(left), None) => tokenizeColumn(left)
```

Comment:
Should be the same as `(None, Some(right))` but then with left?

Reply:
Enabling it for the left side is not required. I did enabled it, but some other tests failed